NitroxModel-Subnautica/DataStructures/GameLogic/NeptuneRocketModel.cs

```diff
@@ -20,6 +20,9 @@ public class NeptuneRocketModel : VehicleModel
         [ProtoMember(3)]
         public bool ElevatorUp { get; set; }
 
+        [ProtoMember(4)]
+        public ThreadSafeCollection<PreflightCheck> PreflightChecks { get; set; }
```

Comment:
ThreadSafeCollection?

Reply:
Works better with protobuf because it's a custom type from us.